### DISCUSSION TITLE
Odia Language Support

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/OSS-DocumentScanner#101] [Ideas]
There are over 35 Milion Odia smartphone users in India, and I would like to contribute in the translation process if you wish to add support for it. Since popular scanner apps do not have odia language, there might be a greater increament in number of consumers too
@farfromrfug what's ur opinion??

**Comment by farfromrefug** [comment]
you can add any new language using weblate. the link is in the readme on github.

**Reply by ghost** [reply]
alright
